### ISSUE TITLE
Improve Accessibility on Main Page

### ISSUE THREAD

**Issue by aeSzelag** [COSC481W-2024Fall/CampQuest#42]
User Story:
Dan is looking for the web application to be more user friendly, so it's easier for him to filter through the campsites based on amenities, camp type, location state, and by name and to clear the search filters with one click and search again. 

Persona:
Dan is a remote worker looking to take his family camping at new campgrounds.

Feature:
Improve site accessibility on the main page, including space between text and submission buttons, text spacing in the drop downs, connecting the checkboxes to their associated text labels, and connect all filter drop downs to the clear button. 

Business Value:
Makes the website easier for more people to use. 

Beta Testing Feedback:
- Mentioned the search bar, some styling in the home page where it could be better fit by adding some spacing.
- Filtering: for accessibility you want to be able to click on the label and not just the check box. Not huge for a small project, but important for real world applications. 
- Pagination: the page numbers are hard to read, not accessible friendly. Can tab through the links on the page, and that is great for accessibility.
- Observed overlap in text boxes when multiple filters were opened.

Task:
- [x] Add margin between text box and search button
- [x] Group the checkbox selections with the associated text
- [x] Highlight page numbers on returned campgrounds
- [x] Make filtering dropdown menu wider to remove overlapping text

Tests:
Clear search button should clear all filtering checkboxes with one click.
Automated testing not required for this task/would not translate for accessibility fixes.

Estimated Workload:
2 days: Impliment changes, run manual testing.

Dependencies: None
